File: src/main/java/no/nav/sbl/dialogarena/mininnboks/consumer/domain/Temagruppe.java
package no.nav.sbl.dialogarena.mininnboks.consumer.domain;

import java.util.List;

import static java.util.Arrays.asList;

public enum Temagruppe {
    ARBD,
    FMLI,
    FDAG,
    HJLPM,
    BIL,
    ORT_HJE,
    OVRG,
    PENS,
    UFRT,
    OKSOS,
    ANSOS;

    public static final List<Temagruppe> GODKJENTE_FOR_INNGAAENDE_SPORSMAAL = asList(ARBD, FMLI, FDAG, HJLPM, BIL, ORT_HJE, PENS, UFRT);
}

File: src/main/java/no/nav/sbl/dialogarena/mininnboks/consumer/domain/Temagruppe.kt
package no.nav.sbl.dialogarena.mininnboks.consumer.domain

import java.util.*

enum class Temagruppe {
    ARBD, FMLI, HJLPM, BIL, ORT_HJE, OVRG, PENS, UFRT, OKSOS, ANSOS;

    companion object {
        val GODKJENTE_FOR_INNGAAENDE_SPORSMAAL = Arrays.asList(ARBD, FMLI, HJLPM, BIL, ORT_HJE, PENS, UFRT)
    }
}
